### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by ZSCDumin** [uber/causalml#164]
![image](https://user-images.githubusercontent.com/19670365/79311793-86814d00-7f30-11ea-828f-8ab2e3032a42.png)

```
learner_x = BaseRRegressor(XGBRegressor(), control_name='control')
learner_x.fit(X=x_train, treatment=treatment_train, y=y_train)
cate_x_no_p = learner_x.predict(X=x_test)

```

Occur error when using BaseRRegressor, how to handle this problem?

**Comment by ThatShelbs** [comment]
Having the same issue. Can run the other Base Regressors just fine but when I run BaseRRegressor I get the same error.

**Comment by ppstacy** [comment]
Hi @ZSCDumin and @ThatShelbs, Thanks for reporting this!

Do you mind provide the full logs for this error? It's a bit difficult now to find out where cause this issue. From the code it didn't provide the propensity score input, so R Learner will automatically run a propensity model to generate the propensity scores.  It looks like this error coming from here.

Could you provide the data which can reproduce this error that I can take a quick look? Thanks!

**Comment by ThatShelbs** [comment]
[https://github.com/ThatShelbs/causalml_baserlearner_issue](Files can now be found here)

The notebook of interest is basic_causalml.ipynb. Included the notebook that generates synthetic data just in case it is needed.

**Comment by jeongyoonlee** [comment]
Thanks, @ThatShelbs for sharing the notebook. The issue has been fixed in PR #170 and working notebook is available in [gist](https://gist.github.com/jeongyoonlee/402f7fd95aa461793579d2d9a8f42f9b). A new version with the fix will be published to PyPI in a few days.
